### PR TITLE
More unit tests

### LINKED ISSUE #72: FactoryGirl rent factory fails lint
When turning on lint for FactoryGirl the factory `:rent` fails linter with

`* rent - Ett fel uppstod: Användaravtal måste anges, Syfte måste anges (ActiveRecord::RecordInvalid)`

### PR DESCRIPTION
**Pull request by davidwessman**
Adding Unit tests for models and controllers.

Fixed #72 with latest commit.

### CODE REVIEW

File: spec/models/rent_spec.rb [hunk 1]
```diff
@@ -2,35 +2,35 @@
 require 'rails_helper'
 
 RSpec.describe Rent, type: :model do
-  subject(:rent) { FactoryGirl.build(:rent) }
+  let(:rent) { FactoryGirl.build(:rent) }
 
   describe :Associations do
-    it { should belong_to(:profile) }
-    it { should belong_to(:council) }
+    it { expect(rent).to belong_to(:profile) }
```
Comment (henrikssn) [inline spec/models/rent_spec.rb]: @jforberg This really shows why `should` > `expect().to`.

**Comment by henrikssn** [comment]
*"Fixed #72 with latest commit."* :+1:

File: spec/models/rent_spec.rb (hunk 1 above)
Comment (Zofz) [inline spec/models/rent_spec.rb]: I might agree with you there ;) 
https://github.com/fsek/web/blob/auth-test/spec/controllers/election/candidates_controller_spec.rb#L79
How would a block expression look with ```should```?

Comment (henrikssn) [inline spec/models/rent_spec.rb]: do
      foo
    end.should be(bar)

Comment (Zofz) [inline spec/models/rent_spec.rb]: Smooth. I will try to start using should then 😄

File: spec/factories/rents.rb [hunk 2]
```diff
@@ -6,6 +6,8 @@
     lastname
     email
     phone
+    disclaimer true
+    purpose "Handla möbler på IKEA"
```
Comment (jforberg) [inline spec/factories/rents.rb]: The hound is right this time. That is a good habit.

**Comment by jforberg** [comment]
This looks good. Just fix the single quotes thing and then go ahead and merge (or rebase).

Simply adding more tests --- that pass --- IMO does not require code review.

**Comment by henrikssn** [comment]
You can fix hound comments automatically with rubocop, i.e. `rubocop -a -c .ruby.style spec/models/rent_spec.rb`. That is very handy :)

File: spec/factories/rents.rb (hunk 2 above)
Comment (henrikssn) [inline spec/factories/rents.rb]: If the hound is not always right then we should reconfigure him ;)

**Pull request closed without merging**